pkg/cli/cli.go

```diff
@@ -983,23 +1001,165 @@ func printConnection(c *command, cvd RemoteCVDLocator, status ConnStatus) {
 	c.Printf("%s/%s: %s\n", cvd.Host, cvd.WebRTCDeviceID, state)
 }
 
-func buildAgentCmdArgs(flags *ConnectFlags, device string) []string {
-	args := []string{
-		ConnectionAgentCommandName,
-		device,
-	}
+func buildAgentCmdArgs(flags *ConnectFlags, device string, connectAgent string) []string {
+	args := []string{connectAgent, device}
 	return append(args, flags.AsArgs()...)
 }
 
-// Handler for the agent command. This is not meant to be called by the user
-// directly, but instead is started by the open command.
+func enableAdbConnection(socketPath string, enable bool) error {
```

Comment:
can we use https://github.com/google/cloud-android-orchestration/blob/main/pkg/cli/adbserver.go? (we need to modify it to accept localfilesystem: though)

Reply:
We can use it, done.